PCSX2/pcsx2
# 3rdparty: Update SDL2 to 2.26.4

### Description of Changes
Targets the latest commit (as of now) on the SDL2 2.26.x branch
https://github.com/libsdl-org/SDL/commit/4fb81fdff92d3816be93a3bcd04e00aff2697b90

### Rationale behind Changes
PCSX2 was unable to shut down SDL. This would cause PCSX2 to freeze when disabling SDL, or leave a zombie task behind when you would close PCSX2.
https://github.com/libsdl-org/SDL/issues/6837

Hopefully this issue is fixed in 2.26.4 🤞 

### Suggested Testing Steps
- Test your peripherals that rely on SDL.
- Close PCSX2 and verify that there is no zombie task taking up CPU time.
- Try turning off SDL in the settings and making sure PCSX2 does not freeze.
